srtcore/api.cpp

```diff
@@ -3307,54 +3257,62 @@ SRT_SOCKSTATUS getsockstate(SRTSOCKET u)
 
 void setloglevel(LogLevel::type ll)
 {
-    CGuard gg(srt_logger_config.mutex);
+    srt_logger_config.lock();
```

Comment:
This change was due to the requirement for thread logging. This part was unnecessary for thread logging, therefore it's replaced with simple lock-unlock.

Reply:
So what is the purpose of this change now? Maybe just revert?

Reply:
The question is still topical.

Reply:
Ok, I'll withdraw it for now.